input/cql/Cataracts2040BCVAwithin90DaysFHIR.cql

```diff
@@ -86,6 +90,13 @@ define "Cataract Surgery Between January and September of Measurement Period":
       and CataractSurgery.performed.toInterval ( ) starts 92 days or more before end of "Measurement Period"
       and CataractSurgery.status = 'completed'
 
+/*
+@QICore6Uplift: 
+  For Conditions of "Burn Confined to Eye and Adnexa", "Choroidal Detachment", "Choroidal Hemorrhage and Rupture", "Cloudy Cornea", "Corneal Edema", "Central Corneal Ulcer", "Injury to Optic Nerve and Pathways",
+    "Open Wound of Eyeball", "Optic Neuritis", "Purulent Endophthalmitis", "Retinal Detachment with Retinal Defect", "Traumatic Cataract", and "Scleritis", 
+    * Update Condition to ConditionEncounterDiagnosis 
+  For the rest of the Conditions not listed above, update Condition to ConditionProblemsHealthConcerns
+*/ 
```

Comment:
Should possibly review with the steward for clinical intent, but as the logic stands this should be a union of ConditionProblemsHealthConcerns and ConditionEncounterDiagnosis

Reply:
So, for the following example, would it be option 1 or option 2? 

Option 1: 
```
with ([ConditionEncounterDiagnosis: "Acute and Subacute Iridocyclitis"]
   union [ConditionProblemsHealthConcerns: "Amblyopia"]
```

Option 2: 
```
with ([ConditionEncounterDiagnosis: "Acute and Subacute Iridocyclitis"]
   union [ConditionProblemsHealthConcerns: "Acute and Subacute Iridocyclitis"]
   union [ConditionEncounterDiagnosis: "Amblyopia"]
   union [ConditionProblemsHealthConcerns: "Amblyopia"]
```